packages/react/src/components/Forms/DatePicker/index.tsx

```diff
@@ -1,176 +1,5 @@
-    datePickerOptions,
-    state,
-    wrapperRef
-  );
-
-  const [isFocused, setIsFocused] = useState(false);
-
-  const labelAsPlaceholder = useMemo(
-    () => !state.isOpen && !state.value,
-    [state.value, state.isOpen]
-  );
-
-  const isDateInvalid = useMemo(
-    () => state.validationState === "invalid" || props.state === "error",
-    [state.validationState, props.state]
-  );
-
-  // onPress props don't exist on the <Button /> component.
-  // Remove it to avoid any warning.
-  const { onPress: onPressToggleButton, ...otherButtonProps } = buttonProps;
-
-  return (
-    <Field {...props}>
-      <div
-        ref={pickerRef}
-        className={classNames("c__date-picker", {
-          "c__date-picker--disabled": disabled,
-          "c__date-picker--invalid": isDateInvalid,
-          "c__date-picker--success": props.state === "success",
-          "c__date-picker--focused":
-            !isDateInvalid && !disabled && (state.isOpen || isFocused),
-        })}
-      >
-        <div
-          className={classNames("c__date-picker__wrapper", {
-            "c__date-picker__wrapper--clickable": labelAsPlaceholder,
-          })}
-          ref={wrapperRef}
-          {...groupProps}
-          aria-label={label}
-          role="button"
-          tabIndex={0}
-          onClick={() => !state.isOpen && state.open()}
-        >
-          {state.value && (
-            <input type="hidden" name={name} value={state.value?.toString()} />
-          )}
-          <div className="c__date-picker__wrapper__icon">
-            <Button
-              {...{
-                ...otherButtonProps,
-                "aria-label": t(
-                  state.isOpen
-                    ? "components.forms.date_picker.toggle_button_aria_label_close"
-                    : "components.forms.date_picker.toggle_button_aria_label_open"
-                ),
-              }}
-              color="tertiary"
-              size="small"
-              className="c__date-picker__wrapper__toggle"
-              onClick={state.toggle}
-              icon={<span className="material-icons">calendar_today</span>}
-              disabled={disabled}
-            />
-          </div>
-          <LabelledBox label={label} labelAsPlaceholder={labelAsPlaceholder}>
-            <div className="c__date-picker__inner">
-              {!labelAsPlaceholder && (
-                <DateField
-                  {...{
-                    ...fieldProps,
-                    onFocusChange: (focus) => setIsFocused(focus),
-                  }}
-                />
-              )}
-            </div>
-          </LabelledBox>
-        </div>
-        {state.isOpen && (
-          <Popover
-            parentRef={pickerRef}
-            onClickOutside={state.close}
-            borderless
-          >
-            <Calendar {...calendarProps} />
-          </Popover>
-        )}
-      </div>
-    </Field>
-  );
-};
+export * from "./types";
```

Comment:
Maybe it is already handled by our previous fix but mind to add export also in the root index.ts :)

Reply:
I have already tested it locally, it should be fine. I will check one more time in the demo app while rebasing my branch.

Reply:
it works fine 😬